Microsoft.Dynamics365.UIAutomation.Sample/UCI/Read/CheckFieldRequired.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Can we move this to a separate folder called validation?

Reply:
Change made